### PR TITLE
[kubespray] Doc in contribute to guide user deploy k8s through kubespray

### PR DESCRIPTION
**Pull request by ydye**
#3618

### CODE REVIEW

**Comment by ydye** [comment]
please review each component's configuration

File: deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+## Configure the amount of pods able to run on single node
+## default is equal to application default
+# kubelet_max_pods: 110
+
+## Support custom flags to be passed to kubelet
+kubelet_custom_flags:
+  - "--eviction-hard="
+  - "--image-gc-high-threshold=100"
+  - "--image-gc-low-threshold=95"
```
Comment (abuccts) [inline deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml]: these flags have been deprecated in kubelet.
> This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.

File: deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+## This network must be unused in your network infrastructure!
+kube_pods_subnet: 10.200.0.0/13
+
+## internal network node size allocation (optional). This is the size allocated
+## to each node on your network.  With these defaults you should have
+## room for 64 nodes with 254 pods per node.
+## Example: Up to 256 nodes, 100 pods per node (/16 network):
+##  - kube_service_addresses: 10.233.0.0/17
+##  - kube_pods_subnet: 10.233.128.0/17
+##  - kube_network_node_prefix: 25
```
Comment (abuccts) [inline deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml]: leading ` -` is not needed

File: deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+## Etcd has a default of 2G for its space quota. If you put a value in etcd_memory_limit which is less than
+## etcd_quota_backend_bytes, you may encounter out of memory terminations of the etcd cluster. Please check
+## etcd documentation for more information.
+## https://etcd.io/docs/v3.4.0/op-guide/configuration/
+etcd_quota_backend_bytes: "8G"
+
+############## nginx configuration ##############
+
+# Requests for load balancer app
```
Comment (abuccts) [inline deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml]: pls keep the same format and use `##`

File: deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+	{%- if docker_version != "latest" and docker_version is version('17.05', '<') %}
+	--graph={{ docker_daemon_graph }} {% if ansible_os_family not in ["openSUSE Leap", "openSUSE Tumbleweed", "Suse"] %}{{ docker_log_opts }}{% endif %}
+	{%- else %}
+	--data-root={{ docker_daemon_graph }} {% if ansible_os_family not in ["openSUSE Leap", "openSUSE Tumbleweed", "Suse"] %}{{ docker_log_opts }}{% endif %}
+	{%- endif %}
+	{%- if ansible_architecture == "aarch64" and ansible_os_family == "RedHat" %}
+	--add-runtime docker-runc=/usr/libexec/docker/docker-runc-current
+	--default-runtime=docker-runc --exec-opt native.cgroupdriver=systemd
+	--userland-proxy-path=/usr/libexec/docker/docker-proxy-current --signature-verification=false
+	{%- endif -%}
```
Comment (abuccts) [inline deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml]: wrong indent in yaml?

File: deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml (hunk 2 above)
Comment (ydye) [reply deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml]: example

Comment (abuccts) [reply deployment/k8sPaiLibrary/kubespray-vars/openpai.yaml]: the example makes it like a list

**Pull request merged**